diff --git a/dist/sentinel.umd.js b/dist/sentinel.umd.js
index 08dcbff40b764a4eabf7990e751c364c4724c885..d5ce9732e2726d9497dc14bcbbcdd155247c3b52 100644
--- a/dist/sentinel.umd.js
+++ b/dist/sentinel.umd.js
@@ -46,7 +46,7 @@ return {
       
       // add stylesheet to document
       styleEl = doc.createElement('style');
-      head.insertBefore(styleEl, head.firstChild);
+      head.append(styleEl);
       styleSheet = styleEl.sheet;
       cssRules = styleSheet.cssRules;
     }